siloLapisTests/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "silolapistests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "generateLapisClient": "npm run runOpenApiGenerator && npm run copyGeneratedFiles && npm run cleanUpGeneratedFiles",
+    "runOpenApiGenerator": "openapi-generator-cli generate -i ../lapis2/lapis-v2-openapi.json -g typescript-fetch -o generated-sources --additional-properties=supportsES6=false",
+    "copyGeneratedFiles": "cp generated-sources/index.ts generated-sources/runtime.ts test/lapisClient && cp -r generated-sources/apis generated-sources/models test/lapisClient",
+    "cleanUpGeneratedFiles": "rm -rf generated-sources"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
```suggestion
```

No need to specify a license here. (If we want so specify the license, we would use GPL 3)

Reply:
Thanks for catching this, I'll get rid of `author` and `keywords`, too.